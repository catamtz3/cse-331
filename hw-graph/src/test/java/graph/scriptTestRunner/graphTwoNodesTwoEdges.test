# Create a graph
CreateGraph twoEdges

#Add two nodes to the graph
AddNode twoEdges a
addNode twoEdges b

#Add one edge between nodes a and b
addEdge twoEdges a b ab
addEdge twoEdges b a ba

# Get the nodes of the graph
ListNodes twoEdges