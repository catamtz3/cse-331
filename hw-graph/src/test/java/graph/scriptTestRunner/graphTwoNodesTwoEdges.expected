# Create a graph
Created graph twoEdges

#Add two nodes to the graph
added node a to twoEdges
added node b to twoEdges

#Add one edge between nodes a and b
added edge ab from a to b in twoEdges

#Add one edge between nodes b and a
added edge ba from b to a in twoEdges

# Get the nodes of the graph
twoEdges contains: a[ab], b[ba]